frontend/src/lib/lemon-ui/icons/icons.tsx

```diff
@@ -512,18 +512,6 @@ export function IconMenu(props: LemonIconProps): JSX.Element {
     )
 }
 
-/** Material Design Menu Open icon. */
-export function IconMenuOpen(props: LemonIconProps): JSX.Element {
```

Comment:
We should remove this icon from https://github.com/posthog/icons if it has already been added & we don't plan on using it again

Reply:
Yup, it doesn't seem to have been added there